security/mkcert/Portfile

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+
+github.setup        FiloSottile mkcert 1.1.0
+github.tarball_from releases
+
+platforms           darwin
+categories          security devel
+license             AS-IS
+installs_libs       no
+
+maintainers         gmail.com:herby.gillot @herbygillot openmaintainer
+
+description         A simple zero-config tool to make locally trusted \
+                    development certificates with any names you'd like
+
+long_description    mkcert is a simple tool for making locally-trusted \
+                    development certificates. It requires no configuration. \
+                    Using certificates from real certificate authorities (CAs)\
+                    for development can be dangerous or impossible (for hosts \
+                    like localhost or 127.0.0.1), but self-signed certificates\
+                    cause trust errors. Managing your own CA is the best \
+                    solution, but usually involves arcane commands, \
+                    specialized knowledge and manual steps. \
+                    \
+                    mkcert automatically creates and installs a local CA in \
+                    the system root store, and generates locally-trusted \
+                    certificates.
+
+platforms           darwin
+
+master_sites        \
+    https://github.com/${github.author}/${github.project}/archive
```

Comment:
You don't need this with the github portgroup.

Reply:
So I've tried before without this, and ports would just try randomly guessing a bunch of fetch URLs before ultimately failing.  I'll try again.

Reply:
Nevermind, this is solved.